Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:xenial
+
+# install latest python and nodejs
+RUN apt-get -y update --fix-missing
+RUN apt-get install -y software-properties-common curl
+RUN add-apt-repository ppa:voronov84/andreyv
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash -
```

Comment:
This is interesting - AFAIK, we've been using the old legacy Node (0.12). Does this work for you?

Under any circumstances, I don't think we should have it here for building Pex files. This node version isn't used for our tests, so it can't really be trusted.